.github/actions/create-version-tag/action.yml

```diff
@@ -0,0 +1,58 @@
+name: Create new version tag
+
+inputs:
+  last-version: 
+    type: string
+    required: true
+
+  version-strategy:
+    type: string
+    required: true
+
+  slack-webhook-url: 
+    type: string
+    required: true
+
+outputs:
+  next-version:
+    description: "The next version given the strategy provided. It has a 'v' prefix on it."
+    value: ${{ steps.next-version.outputs.version }}
+    
+runs:
+  using: "composite"
+  steps:
+    - name: Set package.json current version
+      shell: bash
+      run: |
+          # remove the "v" preffix.
+          current_version=$(echo "${{ inputs.last-version }}" | tr -d 'v')
+
+          # update the package.json file with the current version.
+          jq --arg current_version "$current_version" '.version = $current_version' package.json > tmp.json && mv tmp.json package.json
+
+    - name: Run npm version
+      shell: bash
+      run: "npm version ${{ inputs.version-strategy }} --git-tag-version=false --allow-same-version"
+
+    - name: Get next version
+      shell: bash
+      id: next-version
+      run: |
+        echo "version=v$(jq '.version' package.json --raw-output)" >> "$GITHUB_OUTPUT"
+
+    - name: Create release git tag
+      shell: bash
+      run: git tag -fa ${{ steps.next-version.outputs.version }} -m "${{ github.event.inputs.title }}"
+
+    - name: Push tag to main branch
+      shell: bash
+      # We use --force here to handle any already existing tags - In case we need to re-run this workflow.
+      run: git push --force origin ${{ steps.next-version.outputs.version }}
```

Comment:
This will only push the tag and nothing else, right?

Reply:
Yupp, that's right! 

[Source](https://stackoverflow.com/a/5195913/10745645)